readme.adoc

```diff
@@ -23,15 +21,16 @@ For more information, please check the https://github.com/feedhenry/mobile-secur
 - Account Lockout Policies (via Keycloak)
 - Authentication/Access Control Auditing & Logging (via Keycloak)
 - Authenticated Calls to Protected Endpoints (via Keycloak)
+- Push notifications with UPS server
 
 == Work with Different Backend Services
 
 By default the app will use the backend services that are running on https://security.skunkhenry.com:8443[a dedicated OpenShift cluster] for demonstration purpose. 
 
 You can also configure the app to run against different backend services:
 
-1. Update the URLs in link:secure-ios-app/AppConfig.plist[AppConfig.plist] file.
-2. If you are using HTTPS, update the certificate pinning configuration in link:secure-ios-app/AppDelegate.swift[AppDelegate.swift] file
+1. Update the URLs in link:ios-showcase-template/AppConfig.plist[AppConfig.plist] file.
```

Comment:
Note that this stuff makes no sense now. We use json files. I will leave that for the moment as you may just remove it after

Reply:
Should I just remove the "Work with Different Backend Services" section for now?